src/redux/configureStore.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger, thunk),
+);
+// console.log(store);
```

Comment:
- [ ] Kindly avoid committing commented code to your repo. Comments make your codes look messy.